Makefile

```diff
@@ -573,9 +574,13 @@ ISTIO_SIDECAR_SRC=tools/deb/istio-start.sh \
                   security/tools/deb/istio-auth-node-agent.service \
```

Comment:
I think the security/ prefix should be deleted from this line.

Reply:
Fixed